docs/java/how-to-develop-a-workflow-definition-in-java.md

```diff
@@ -0,0 +1,186 @@
+---
+id: how-to-develop-a-workflow-definition-in-java
+title: How to develop a Workflow Definition in Java
+sidebar_label: Workflow Definition
+description: In the Temporal Java SDK programming model, a Workflow is a class which implements a Workflow Interface.
```

Comment:
Per https://developers.google.com/style/pronouns?hl=en#relative-pronouns, changed "which" to "that".

I'm fairly certain that we don't claim the term "Workflow Interface," so we should be safe lowercasing "interface." Someone please correct me if I'm wrong.

```suggestion
description: In the Temporal Java SDK programming model, a Workflow is a class that implements a Workflow interface.
```

Reply:
Changed this and updated it in a few other places also.